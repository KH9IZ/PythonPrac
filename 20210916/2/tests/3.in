[-100, 100, +100, 321123, 456654, -909]
