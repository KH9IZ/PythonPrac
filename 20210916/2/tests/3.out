[-909, -100, 100, 100, 321123, 456654]
